### PR TITLE
Document the issue with using properly formatted timeoutForDataDownload

### PR DESCRIPTION
**Pull request by ashahba**

### CODE REVIEW

File: docs/user.md [hunk 1]
```diff
@@ -338,9 +333,21 @@ source types is given below.
...
+## Known issues
+- The NFS server ip and path are not validated, so please ensure that the servers are routable and paths are valid prior
+to the creation of the VolumeManager CR.
+In case an invalid `server` or `path` is used, Kubernetes publishes an event similar to the
+following during first attempt to use the PVC:
+```
+Unable to mount volumes for pod : timeout expired waiting for volumes to attach/mount for pod ...
+```
+
+- `timeoutForDataDownload` value should be of valid format as described [here](https://golang.org/pkg/time/#ParseDuration)
```
Comment (balajismaniam) [inline docs/user.md]: This is not an issue. Also, it is already documented in https://github.com/kubeflow/experimental-kvc/blob/master/docs/user.md#types-of-sources.

**Comment by ashahba** [comment]
@balajismaniam it is  not clearly documented that `one bad value causes all volumemangers to hang`.
That is the point of this PR 🙂 
Here is one:
```yaml
apiVersion: kvc.kubeflow.org/v1
kind: VolumeManager
metadata:
  name: scratchpad-notimeout-hangs
  namespace: ashahba
spec:
  volumeConfigs:
    - id: "scratchpad-notimeout-bad-03"
      sourceType: "S3"
      sourceURL: "SOME_S3_PATH"
      accessMode: "ReadWriteOnce"
      endpointURL: "https://storage.googleapis.com"
      capacity: 100Mi
      replicas: 2
      labels:
        key1: scratchpad-notimeout-hangs
      options:
        awsCredentialsSecretName: gcs-creds
        timeoutForDataDownload: 10
```

Once this is deployed all attempts to create volumemanager is going to fail.

```I0518 16:28:32.451026       1 reflector.go:240] Listing and watching *v1.VolumeManager from github.com/kubeflow/experimental-kvc/pkg/client/informers/externalversions/factory.go:60
E0518 16:28:32.453485       1 reflector.go:205] github.com/kubeflow/experimental-kvc/pkg/client/informers/externalversions/factory.go:60: Failed to list *v1.VolumeManager: v1.VolumeManagerList.Items: []v1.VolumeManager: v1.VolumeManager.Spec: v1.VolumeManagerSpec.VolumeConfigs: []v1.VolumeConfig: v1.VolumeConfig.Options: ReadString: expects " or n, but found 1, error found in #10 byte of ...|ownload":10},"replic|..., bigger context ...|SecretName":"gcs-creds","timeoutForDataDownload":10},"replicas":2,"sourceType":"S3","sourceURL":"s3:|...```

**Comment by balajismaniam** [comment]
@ashahba It is already documented elsewhere. 
During offline discussions you mentioned that the controller crashed when you added incorrect time formats. Please document or open an issue for that. I'm closing this PR for now.

**Comment by ashahba** [comment]
Sounds good @balajismaniam but I tried to document it here in this PR:
https://github.com/kubeflow/experimental-kvc/pull/50/files#diff-2872b51e38bcd053c0e88c4fe5c28fd2R347

**Comment by balajismaniam** [comment]
@ashahba That is already document. To be clear, I'm asking you to document the controller crash error using an issue or PR fixing that issue.

**Pull request closed without merging**